### PR TITLE
Fixes #8670 - Makes explicit the example reference

### LINKED ISSUE #8670: Issue with "async function": ambiguity at end
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

Was unclear which of the two preceding code snippets is being referred to by "after the return keyword," and that's a crucial thing here.

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#rewriting_a_promise_chain_with_an_async_function

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/statements/async_function`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/statements/async_function/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

### PR DESCRIPTION
**Pull request by himanshugarg**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

Fixes #8670

> What was wrong/why is this fix needed? (quick summary only)

The statement had two code snippets above it in the same section. The OP suggested we make explicitly which one.

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/javascript/reference/statements/async_function/index.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ async function getProcessedData(url) {
 }
 ```
 
-In the above example, notice there is no `await` statement after the
+In the single async function above, notice there is no `await` statement after the
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/statements/async_function/index.md:313]:
```suggestion
In the second example, notice there is no `await` statement after the
```

...seems clearer?

**Review by wbamberg — COMMENTED** [review]
I made a suggested change. In particular I think "**single** async function" is confusing, because they are both single functions, just the first has multiple returns.

File: files/en-us/web/javascript/reference/statements/async_function/index.md (hunk 1 above)
Comment (himanshugarg) [reply files/en-us/web/javascript/reference/statements/async_function/index.md]: Yes. Thanks. I reused the phrase from earlier for the same effect but this is much better.

**Review by wbamberg — APPROVED** [review]
👍 thanks for the update @himanshugarg !

**Pull request merged**